beets/importer.py

```diff
@@ -170,6 +170,22 @@ def history_get():
     return state[HISTORY_KEY]
 
 
+def calculate_quality_score(item, is_album):
```

Comment:
Extremely small bikeshed note: I might shorten this name to simply `quality_score`.